ts/Dashboards/Actions/Resizer.ts

```diff
@@ -181,7 +181,7 @@ class Resizer {
         const minHeight = options.styles.minHeight;
         const snapWidth = this.options.snap.width || 0;
         const snapHeight = this.options.snap.height || 0;
-        const dashboardContainer = this.editMode.dashboard.container;
+        const dashboardsContainer = this.editMode.dashboard.container;
```

Comment:
IMO this should be singular because we are referring to a specific instance of the Dashboards.

Reply:
Good point, I've started it before renaming.